imageai/Detection/Custom/__init__.py

```diff
@@ -507,7 +508,7 @@ def _create_callbacks(self, saved_weights_name, model_to_save):
 
         checkpoint = CustomModelCheckpoint(
             model_to_save=model_to_save,
-            filepath=saved_weights_name + 'ex-{epoch:02d}--loss-{loss:.2f}.h5',
+            filepath=saved_weights_name + 'ex-{epoch:03d}--loss-{loss:8.3f}.h5',
```

Comment:
Filenames have now had spaces introduced in the loss part 
'detection_model-ex-003--loss-   6.306.h5'
would suggest {loss:08.3f}
otherwise  the changes are functioning OK on the hololens example
TEST OK  with path to models  and a single model file name  
trainer.evaluateModel(model_path="hololens/models"
trainer.evaluateModel(model_path="hololens/models/detection_model-ex-003--loss-   6.306.h5"